src/main/kotlin/org/jitsi/jibri/capture/ffmpeg/util/OutputParser.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.jitsi.jibri.capture.ffmpeg.util
+
+import org.jitsi.jibri.util.decimal
+import org.jitsi.jibri.util.oneOrMoreNonSpaces
+import org.jitsi.jibri.util.zeroOrMoreSpaces
+import java.util.regex.Pattern
+
+const val ENCODING_KEY = "frame"
```

Comment:
Perhaps some docs on what these are and how they are used?

If this is just a grep for "frame" perhaps that's a little too wide and could match lines that you didn't intend to?

Reply:
yeah i'll add some docs here.  so far there hasn't been an issue with regards to catching something we didn't...but this is also used as a key from a string parsed from a `key=value` substring, so really this will only be matched against a `frame=XXX` type string.